### ISSUE TITLE
[css-color-hdr] Define "media white" better

### ISSUE THREAD

**Issue by jyasskin** [w3c/csswg-drafts#11346]
https://drafts.csswg.org/css-color-hdr/ defines "media white" as all of
* the color of a normal white background, or of white text on a dark background
* 80 cd/m²
* 200 cd/m²
* 203 cd/m²
* whatever the user sets it to

I've also heard a definition that it's meant to be the color of a white piece of paper (no idea what reflectance) held next to the monitor, and @Myndex says in https://github.com/w3c/csswg-drafts/issues/3435#issuecomment-733434472 that it should be (or default to) 5x the ambient surrounding light.

It'd be good to pick a single definition for the web's purposes. The conflicting definitions from sRGB and rec2100-pq should be noted, and the spec should say how to adapt them to the user's choice of luminosity (with results for PQ depending on the resolution of #10460).

**Comment by svgeesus** [comment]
I agree that there are many different definitions, as we reference a lot of different specifications of different ages and assumptions.

sRGB, from the 1990s and aimed at the CRT monitors of the time, picked 80 cd/m². Adobe RGB [picked 160 cd/m²](https://drafts.csswg.org/css-color-4/#predefined-a98-rgb) and ProPhoto RGB [picked 160.0 to 640.0 cd/m²](https://drafts.csswg.org/css-color-4/#predefined-prophoto-rgb). Display P3, derived from DCI PC (which uses 48 cd/m², appropriate for a digital projector in a totally dark movie theatre) went for sRGB compatibility and thus selected 80 cd/m².

Those examples are all SDR, meaning that users are expected to adjust their screen brightness up or down to suit their comfort and thus, that all luminances are relative to the actual luminance of media white, which is placed at 100%. This also means that there wasn't a lot of attention paid to the nominal absolute luminance level of white. And by and large, it didn't matter much.

The introduction of HDR changes things in a number of ways:

1. a comfortable white that could be used, for example, for subtitles or for a white background is no longer "the maximum value"
2. the maximum levels are uncomfortable to look at over a large area
3. the maximum levels can't even be _produced_ except over a small area and a limited time (power consumption, heating)
4. the adaptation state of the eye, derived from the whole screen contents, becomes more important
5. there is a need to display SDR and HDR content together (overlays, subtitles, etc)

and thus the concept of media white and the need to pick a value that works with HDR content. 80 cd/m² looks weirdly dim. 203 cd/m² looks good and is widely recommended. 2000 cd/m² (or whatever the whole-screen brightest white happens to be) is completely unusable.

**Comment by svgeesus** [comment]
All of which is to say that we can _recommend_, but not _require_, a media white of 203 cd/m², for intermediate calculations such as colorspace conversion; and that user adjustments and tone mapping mean that the actual measured value will not be 203.
